chaoskube/chaoskube_test.go

```diff
@@ -40,6 +41,7 @@ func (suite *Suite) TestNew() {
 		excludedWeekdays   = []time.Weekday{time.Friday}
 		excludedTimesOfDay = []util.TimePeriod{util.TimePeriod{}}
 		excludedDaysOfYear = []time.Time{time.Now()}
+		minimumAge         = time.Duration(0)
```

Comment:
I like to put a specific value here to ensure it's really tested. I think `time.Duration(0)` is the default value for this type. How about using `time.Duration(42)` as the value 😃